Resources/public/js/views/confirmbox/ez-draftconflictview.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-draftconflictview', function (Y) {
+    "use strict";
+
+    Y.namespace('eZ');
+
+    var CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_ROW = '.ez-draft-conflict-list-row',
+        SELECTOR_CONTENT = '.ez-draft-conflict-list-content',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_ROW] = {tap: '_uiShowRowOptions'};
```

Comment:
you are duplicating some code from DashboardBlockAsynchronousView, it would be nice to extract the line selection logic in a dedicated extension applied to DashboardBlockAsynchronousView and DraftConflictView